packages/authentication/src/decorator.ts

```diff
@@ -3,16 +3,10 @@
 // This file is licensed under the MIT License.
 // License text available at https://opensource.org/licenses/MIT
 
+import {AuthenticationMetadata} from '@loopback/core';
```

Comment:
Moving `AuthenticationMetadata` to `@loopback/core` does not look right to me. One of the hard lessons we have learned in LoopBack 1.x/2.x/3.x is that we need to keep the core as small as possible and leaving as much of the authentication-related code in a different module. That way it's easy for applications to use a completely different auth/auth implementation, or perhaps use their own modified fork of `@loopback/authentication` while keeping the rest of their dependencies vanilla loopback.

cc @ritch